### DISCUSSION TITLE
简悦第三方适配源汇总（内含使用方式）

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2287] [站点适配]
## 如何配置

> 直接复制 https://gist.github.com/Kenshin/aa3188b22f2d5dcd130610ce41cf6a7b/raw/website_list.json

注意：

1. 此文件在 https://gist.github.com/Kenshin/aa3188b22f2d5dcd130610ce41cf6a7b
2. 此方式提供的即为永久且为最新的 `website_list.json`

感谢简悦社区用户 **moi** 的帮助~

## 如何使用

打开 **简悦 → 选项页 → 站点管理**，并将上步得到的 URL 复制到此后点击，如下图

![image](https://user-images.githubusercontent.com/81074/115182322-5bfe3d00-a10c-11eb-8aee-50305eb6060a.png)

当出现类似下图的提示：（请忽略具体的数字）

![image](https://user-images.githubusercontent.com/81074/115182353-6e787680-a10c-11eb-9c75-f5d4ed2274e1.png)

## 社区适配源

> 有一些站的适配属于较个人喜好，如：[少数派文章页面，增加捕捉题图、作者和发布日期](https://gist.github.com/binsee/55acad05b921ba79ac6695a9463f563c#file-sspai-com-json)，[为知乎问答页面增加适配问题为页面描述](question.zhihu.com.json) 等更个性化的适配规则。

请直接在 https://gist.github.com/binsee/55acad05b921ba79ac6695a9463f563c 查看以及使用。（使用方法如上）

**Comment by hellozgl** [comment]
请问这些第三方适配规则需要定期在简悦的偏好设置中，做定期手动更新吗。还是会自动更新呢？

**Reply by Kenshin** [reply]
第三方适配源是需要定期更新的，它不会跟主适配源一样，有自动更新的功能。

但它本身来说更新的频率较低，所以不定期手动点点就行。

**Comment by whitewatercn** [comment]
https://gist.github.com/Kenshin/aa3188b22f2d5dcd130610ce41cf6a7b?permalink_comment_id=5831414#gistcomment-5831414 
@Kenshin 由于gist无法申请pr，请kenshin手动修改相关gist，或者新建一个gist？或者新建一个仓库专门放相关第三方适配源，方便pr管理？

**Reply by Kenshin** [reply]
> 新建一个仓库专门放相关第三方适配源，方便pr管理？

okay 可以考虑后者 Create 后会在这里通知你 😄 🙏  👍
